app/models/image.rb

```diff
@@ -1,2 +1,3 @@
 class Image < ActiveRecord::Base
+  belongs_to :imageable, polymorphic: true
```

Comment:
in order to get these polymorphic associations (this image one along with votes, and reviews), you'll need to add the two fields to the database. so in this specific case, you'd need `imageable_id` and `imageable_type` in your `images` table

your syntax is great though 👍